### PR TITLE
Feature/remove ws cache

### PR DESCRIPTION
**Pull request by dbmeneses**
Missing ITs (want to see first how it goes in travis)
Will merge both commits into one with ticket name after review.

### CODE REVIEW

File: sonar-scanner-engine/src/main/java/org/sonar/batch/bootstrapper/Batch.java [hunk 1]
```diff
@@ -159,10 +159,10 @@ private RuntimeException handleException(RuntimeException t) {
 
   /**
    * @since 5.2
+   * @deprecated since 5.6
    */
   public Batch syncProject(String projectKey) {
```
Comment (SonarTech) [inline sonar-scanner-engine/src/main/java/org/sonar/batch/bootstrapper/Batch.java]: MAJOR Add the missing @Deprecated annotation. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
INFO Do not forget to remove this deprecated code someday. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)

File: sonar-scanner-engine/src/test/java/org/sonar/batch/repository/DefaultGlobalRepositoriesLoaderTest.java [hunk 2]
```diff
@@ -19,60 +19,31 @@
...
 
-import static org.assertj.core.api.Assertions.assertThat;
+import java.io.StringReader;
+
 import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.verifyNoMoreInteractions;
-import static org.mockito.Mockito.when;
 
 public class DefaultGlobalRepositoriesLoaderTest {
```
Comment (SonarTech) [inline sonar-scanner-engine/src/test/java/org/sonar/batch/repository/DefaultGlobalRepositoriesLoaderTest.java]: MAJOR Add some tests to this class. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2187)

**Comment by SonarTech** [comment]
SonarQube analysis reported 3 issues
* MAJOR 2 major
* INFO 1 info

Watch the comments in this conversation to review them.

**Comment by henryju** [comment]
LGTM

**Pull request merged**